docs/source/fundamentals/interfaces.rst

```diff
@@ -3,7 +3,7 @@
 Interfaces
 ==========
 
-.. include:: ../snippets/slack_support.rstinc
+.. include:: ../snippets/_slack_support
```

Comment:
recommend using the substitution defs shows here: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#substitution-definitions

include has some security issue in that it pastes the whole file. these substitutions go thr the rst compiler and are safer to use. also easier